content/client-server-api/modules/receipts.md

```diff
@@ -22,33 +22,68 @@ that the user had read all events *up to* the referenced event. See the
 [Receiving notifications](#receiving-notifications) section for more
 information on how read receipts affect notification counts.
 
+{{< added-in v="1.4" >}} Read receipts exist in three major forms:
```

Comment:
MSC3771 doesn't specify that unthreaded/threaded applies only to *read* receipts, but there isn't currently any other type of receipts...

Reply:
yea, we're very slowly moving this module to talk about only read receipts at the moment (at least until another receipt shows up)

Reply:
Makes sense, just wanted to ensure this was on purpose.